libs/schematic/api-description/build/openapi.yaml

```diff
@@ -94,6 +94,28 @@ paths:
                 $ref: '#/components/schemas/AttributesPage'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /components/{componentLabel}/parents:
```

Comment:
As noted elsewhere this is not a component specific endpoint and can be used for any node in the graph, would likely only return viable results for attributes, but if an attribute was also a valid value results would be returned if the attributes would have dependencies.